servers/publikator/.env.example

```diff
@@ -33,3 +33,9 @@ MAILCHIMP_REPLY_TO=kontakt@republik.ch
 #VIMEO_APP_ACCESS_TOKEN=
 
 #YOUTUBE_APP_KEY=
+
+#############
+# Maintenance Flags
+#############
+# PUBLIKATOR_PUBLISH_DISABLE=true
+# PUBLIKATOR_UNPUBLISH_DISABLE=true
```

Comment:
IMHO one ENV should be more than enough, and it shouldn't be prefixed with `PUBLIKATOR`. I propose: `DISABLE_PUBLISH`